ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/EthSchedulerShutdownTest.java

```diff
@@ -85,8 +87,8 @@ public void shutdown_txWorkerShutsDown() throws InterruptedException {
     final MockEthTask task1 = new MockEthTask(1);
     final MockEthTask task2 = new MockEthTask();
 
-    ethScheduler.scheduleTxWorkerTask(task1::executeTask);
-    ethScheduler.scheduleTxWorkerTask(task2::executeTask);
+    ethScheduler.scheduleTxWorkerTask(BoundedTimedTask.fromRunnable(task1::executeTask));
```

Comment:
Why not move this wrapping into `scheduleTxWorkerTask`?

Reply:
Done.